utils/whiteouts/whiteouts.go

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright (C) 2022 Renesas Electronics Corporation.
+// Copyright (C) 2022 EPAM Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package whiteouts
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/aoscloud/aos_common/aoserrors"
+	"golang.org/x/sys/unix"
+)
+
+/***********************************************************************************************************************
+* Consts
+***********************************************************************************************************************/
+
+const (
+	whiteoutPrefix    = ".wh."
+	whiteoutOpaqueDir = whiteoutPrefix + whiteoutPrefix + ".opq"
```

Comment:
You mask whiteoutOpaqueDir by whiteoutPrefix. If only whiteoutPrefix has to be changed then whiteoutOpaqueDir will be changed implicitly.

Reply:
Split declaration independent